ros_buildfarm/common.py

```diff
@@ -634,6 +634,21 @@ def get_downstream_package_names(pkg_names, dependencies):
     return downstream_pkg_names
 
 
+def get_cached_package_manifests(dist):
```

Comment:
The function name might be misleading. The manifests don't seem to be fetched from the cache but queried through the manifest providers, no?